### ISSUE TITLE
Changes are not displayed in the table when you have multiple key fields in the Entity

### ISSUE THREAD

**Issue by deepakm3** [cap-js/change-tracking#115]
The change tracking is not working when we have multiple key fields for an entity, Is this a limitation ?

Example below works, The changes are updated and displayed in the object page

entity Books {
  key ID : Integer;
  title  : String;
  stock  : Integer;
}

Whereas when you have multiple key field the The changes are not displayed in the object page

entity Books {
  key ID : Integer;
  key title  : String;
  stock  : Integer;
}

**Comment by nkaputnik** [comment]
Yes, this is a known limitation as of now, but we are working on fixing this in the next release.

Best,
Nick

**Comment by deepakm3** [comment]
Could you please let me know the planned release date for the fix?
Thanks,
Deepak

**Comment by nkaputnik** [comment]
Unfortunately, no.
This is an open-source project, which is maintained by us in our spare time, so we do not plan any releases, we do them, when we have enough spare time and enough changes to warrant one...

This feature is change is rather complex in nature and requires extensive testing.

Best,
Nick

**Comment by sammok83** [comment]
@nkaputnik I echo @deepakm3 . Would also need this.

**Comment by nkaputnik** [comment]
There is an open PR aiming to solve this: #117 
Plan is to finalise and test this in November, since the contributor is off work for this whole month.

**Comment by Sv7enNowitzki** [comment]
Merge this issue into #130  because the problem is the same.
